src/bootstrap/lib.rs

```diff
@@ -65,9 +65,15 @@
 //! also check out the `src/bootstrap/README.md` file for more information.
 
 #![deny(warnings)]
+#![feature(associated_consts)]
+#![feature(core_intrinsics)]
```

Comment:
The intrinsic only gives you an integer. It can be safe but it's also useless, unless combined with `unsafe` code to build an `Any`-like abstraction.